packages/main/src/RPA/Robocorp/Process.py

```diff
@@ -444,11 +459,12 @@ def list_process_runs_in_workspace(
     @keyword(tags=["process", "get", "runs"])
     def get_process_run_status(
         self, process_run_id: str, process_id: Optional[str] = None
-    ):
+    ) -> json:
```

Comment:
I think that this incorrect type when returning json objects. Better and a bit generic from json structure would be Dict[str, Any]. The json is package name.